README.md

```
@@ -62,7 +64,7 @@
 
 | **이진호** | **김원호** | **김혜민** | **서영진** | **유다연** |
 | :------: |  :------: | :------: | :------: | :------: |
-| [<img src="https://github.com/user-attachments/assets/e0f9fb2b-70cd-4f46-87d9-3bc58512ec91" height=150 width=150> <br/> @treasure-sky](https://github.com/treasure-sky) | [<img src="https://github.com/user-attachments/assets/458ff5bd-889b-4cb3-a8ff-aa641f0c6819" height=150 width=150> <br/> @kmwh](https://github.com/kmwh) | [<img src="https://github.com/user-attachments/assets/a145d137-8039-4445-a776-72b80bf3311f" height=150 width=150> <br/> @hyemomo](https://github.com/hyemomo) | [<img src="https://github.com/user-attachments/assets/823d2a52-ece4-49e2-b52d-bfd958c7a83e" height=150 width=150> <br/> @seoyoungjin23](https://github.com/seoyoungjin23) | [<img src="https://github.com/user-attachments/assets/e08aa2cd-b32c-4ae6-aaac-1664decece43" height=150 width=150> <br/> @daaoooy](https://github.com/daaoooy) |
+| [<img src="https://github.com/daaoooy/boost-assets/blob/main/assets/members-avatar/be-ljh.png" height=150 width=150> <br/> @treasure-sky](https://github.com/treasure-sky) | [<img src="https://github.com/daaoooy/boost-assets/blob/main/assets/members-avatar/be-kwh.png" height=150 width=150> <br/> @kmwh](https://github.com/kmwh) | [<img src="https://github.com/daaoooy/boost-assets/blob/main/assets/members-avatar/fe-khm.png" height=150 width=150> <br/> @hyemomo](https://github.com/hyemomo) | [<img src="https://github.com/daaoooy/boost-assets/blob/main/assets/members-avatar/bg-syj.png" height=150 width=150> <br/> @seoyoungjin23](https://github.com/seoyoungjin23) | [<img src="https://github.com/daaoooy/boost-assets/blob/main/assets/members-avatar/fe-ydy.png" height=150 width=150> <br/> @daaoooy](https://github.com/daaoooy) |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)